src/libraries/System.Runtime/tests/System/DoubleTests.cs

```diff
@@ -1615,5 +1615,16 @@ public static void TanPiTest(double value, double expectedResult, double allowed
             AssertExtensions.Equal(-expectedResult, double.TanPi(-value), allowedVariance);
             AssertExtensions.Equal(+expectedResult, double.TanPi(+value), allowedVariance);
         }
+
+        [Theory]
+        [InlineData(+1e+20, 1e+20, 0.0)]
+        [InlineData(-1e+10, 1e+10, 0.0)]
+        [InlineData(-1e+20, 1e+20, 0.0)]
+        public static void Regression75651(double value, double expectedResult, double allowedVariance)
```

Comment:
Nit: Can we name the test after what it's testing rather than the associated issue number?

Reply:
Not sure of a good name for this test as it's covering an explicit path that is otherwise an "implementation detail" of the function.

Basically its testing that the result is correct when the difference between the two inputs is "large".

Reply:
TestValuesWithLargeDifferences(..) 🙂

Reply:
I'm not sure that's really more descriptive and its more likely overall to conflict with other tests that do the same.

Personally, I prefer the name `Regression75651` because if the test fails, it tells me exactly where to look for more information and then I have the contextual history on the bug, what its meant to be covering, and why it was added. Alternatively, `TestValuesWithLargeDifferences` only tells me what the test is covering, not why the test is here and separate from the regular `HypotTests`.

Reply:
> it tells me exactly where to look for more information and then I have the contextual history

To you, today.

That context decreases over time. What is this test validating, for example?
https://github.com/dotnet/runtime/blob/58f5c95085b2eb56800b87f466183b1448d33b3d/src/libraries/System.Data.Common/tests/System/Data/DataRelationTest2.cs#L500
Even if someone can figure it out, it takes non-trivial time to do so.

A comment with a link to the github issue is fine; that's even easier for someone to follow, provides concrete context as to the associated repo, is clickable in some browsers, etc.

My preference is to pick as descriptive name as possible, and include the link in a comment as context. If you'd had the foresight to test this when originally writing tests, what would it have been called? Name it that :-) 

Anyway, that's my preference. Ultimately it's up to you.

Reply:
> If you'd had the foresight to test this when originally writing tests, what would it have been called?

This would have just been another theory entry on HypotTests originally since it’s really just validating another basic input/output scenario.

The Regression##### is only minorly meaningful information to help indicate it was a missed scenario originally

Reply:
If it were me, then, I'd just add it as another theory entry now, and if desirable, suffix it with a comment link to the github issue.

Reply:
Sounds good. I’ll get that updated in a little bit